packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6361,3 +6861,214 @@ Property details
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- |
+| capabilities            | capabilites     | [Capabilities Object](#capabilties-object)                                                                  | FALSE    | FALSE   | TRUE    |           |
+| description            | description      | String  | FALSE    | FALSE   | TRUE    |           |
+| _links           | discoverable resources related to the app user     | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)     | TRUE    | FALSE   | TRUE    |           |
+| name              | identifiying name     | `USER_PROVISIONING`  | FALSE    | FALSE  | TRUE    |           |       |      |
+| status            | status    | `ENABLED`, `DISABLED`    | FALSE    | FALSE   | TRUE    |      `DISABLED`  |
+
+##### Capabilties object
+
+The Capabilities object determines the capabilities of an app feature.
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
+| create            | determines whether the Okta creates or links a user in the application when assigning the app to a user in Okta | [Create Object](#create-object)  | TRUE    | FALSE   | FALSE    |
+| update            | determines whether the updates to a user's profile will be pushed to the application | [Update Object](#update-object)  | TRUE    | FALSE   | FALSE    |
+
+###### Create object
+
+The Create object is composed of a single setting that determines whether the application creates or links a user in the application when assigning the app to a user in Okta.
+
+```json
+{
+  "lifecycleCreate": {
+    "status": "DISABLED"
+  }
+}
+```
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- |
+| lifecycleCreate            | setting that determines whether the updates to a user in Okta will be update a user in the application   | [Lifecycle Create Setting Object](##lifecycle-create-setting-object)      | TRUE    | FALSE   | FALSE    |           |
+
+###### Update object
+
+The Create object is composed of multiple settings that determine whether the updates to a user in Okta will update a user in the application.
```

Comment:
The Create object is composed of multiple settings that specify whether the updates to a user in Okta then update the user in the application.